ios/platform/README.md

```diff
@@ -1,9 +1,9 @@
-Files in this folder are copied into the iOS build directory
+Files in this folder are copied directory into the iOS build directory
 when the iOS app is compiled:
 
     <project-dir>/build/iphone
 
-You can place files such as asset catalog files and storyboards in this
+You can place files such as asset catalog files, .framework files and storyboards in this
 directory.
 
 Files in this directory are copied directly on top of whatever files are already
```

Comment:
```suggestion
Files in this folder are copied directly on top of whatever files are already
```
Or however you want to clean this up. GitHub won't let me `suggest` the "directory" in line 2.